python/lsst/sconsUtils/scripts.py

```diff
@@ -168,13 +168,17 @@ def finish(defaultTargets=DEFAULT_TARGETS,
         if "tests" in [str(t) for t in BUILD_TARGETS]:
             testsDir = pipes.quote(os.path.join(os.getcwd(), "tests", ".tests"))
             checkTestStatus_command = state.env.Command('checkTestStatus', [], """
-                @ if [ -d %s ]; then \
-                      nfail=`find %s -name \*.failed | wc -l | sed -e 's/ //g'`; \
+                @ if [ -d {0} ]; then \
+                      nfail=`find {0} -name \*.failed | wc -l | sed -e 's/ //g'`; \
                       if [ $$nfail -gt 0 ]; then \
+                          echo "Failed test output:" >&2; \
+                          find {0} -name \*.failed -exec cat {{}} \; >&2; \
```

Comment:
The `>&2;` has essentially no effect.  It probably needs to be something like `find {0} -name \*.failed -exec cat {} \; | cat >&2` (may require some fiddling).